Vagrantfile

```diff
@@ -3,20 +3,18 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
+# rubocop:disable Metrics/BlockLength
 Vagrant.configure('2') do |config|
-  config.vm.box = 'omu/debian-stable-server'
+  config.vm.define 'dev' do |dev|
+    dev.vm.box = 'omu/debian-stable-server'
 
-  config.vm.network 'forwarded_port', guest: 3000, host: 3000
+    dev.vm.network 'forwarded_port', guest: 3000, host: 3000
 
-  config.vm.provider 'virtualbox' do |vb|
-    vb.memory = '2048'
-  end
-
-  config.vm.provider :lxc do |lxc|
-    lxc.customize 'cgroup.memory.limit_in_bytes', '2048M'
-  end
+    dev.vm.provider :lxc do |lxc|
+      lxc.customize 'cgroup.memory.limit_in_bytes', '2048M'
+    end
 
-  config.vm.provision 'shell', privileged: false, inline: <<-SHELL
+    dev.vm.provision 'shell', privileged: false, inline: <<-SHELL
```

Comment:
bir sonraki refaktörde belki bu kısmı da dış bir betiğe alabiliriz, github takımının da kullandığı şöyle bir düzenlemeyle: https://github.com/github/scripts-to-rule-them-all  tek fark şu olabilir, script dizini yerine bin dizini kullanılmalı.